Woof/Woof/View/SitterFlow/EditSitterInformationView.swift

```diff
@@ -0,0 +1,119 @@
+import SwiftUI
+
+/// A view to edit the sitter information.
+struct EditSitterInformationView: View {
+    // MARK: - Internal interface
+
+    /// The name of the sitter.
+    @Binding var name: String
+
+    /// The surname of the sitter.
+    @Binding var surname: String
+
+    /// The phone of the sitter.
+    @Binding var phone: String
+
+    /// The additional information about the sitter, like his experience with dogs, favourite places for walks,
+    /// special skills, certificates, etc.
+    @Binding var bio: String
+
+    /// The price per hour for walking charged by the sitter.
+    @Binding var pricePerHour: String
+
+    var body: some View {
+        VStack {
+            Text(title)
+            TextField(nameTextFieldText, text: $name)
+            TextField(surnameTextFieldText, text: $surname)
+            TextField(phoneTextFieldText, text: $phone)
+                .keyboardType(.phonePad)
+            TextField(priceTextFieldText, text: $pricePerHour)
+                .keyboardType(.decimalPad)
+            TextEditorWithPlaceholder(message: $bio, placeholder: bioInformationPlaceholderText)
+                .cornerRadius(AppStyle.UIElementConstant.cornerRadius)
+
+            if name.isEmpty {
+                Text(mandatoryTextFieldText)
+            }
+        }
+        .padding()
+        .foregroundColor(.App.purpleDark)
+        .textFieldStyle(.roundedBorder)
+        .background(Color.App.purpleLight)
+        .cornerRadius(AppStyle.UIElementConstant.cornerRadius)
+    }
+}
+
+// MARK: - Private interface
+
+private let title = "Edit your information"
+private let bioInformationPlaceholderText = "Enter information about yourself"
+private let nameTextFieldText = "Enter your name"
```

Comment:
```suggestion
private let nameTextFieldPlaceholderText = "Enter your name"
```
It'll be consistent with the name of the constant on the line above and clearer. See the rest of code.

Reply:
I choose the name `namePlaceholderText`. What do you think?